examples/hello/lib/hello/app.rb

```diff
@@ -0,0 +1,66 @@
+require "hello/actions"
+
+module Hello
+  class App
+    class Router
```

Comment:
re: [this outdated comment](https://github.com/blaix/wren/pull/1#discussion_r32370757):

> Once it's working right I will move this to lib/wren/router.

Done in 6b108937d6fe0a722f376ea7e94ed8a309e65442

> I will also want to start driving some `[resource_name]_path` methods with unit tests.

Not a concern right now.